helloworld/README.md

```diff
@@ -4,15 +4,33 @@ The model consists of the following configurations:
 - `helloworld-topology.yaml` - This defines the network topology for the hwlloworld model.
 
 The model uses the following backing images:
-- `ubuntu.qc2` - the ubuntu image can be built using `phenix image` with the following commands:
-    ```
-    phenix image create /phenix/vmdb2/scripts/ubuntu --format qcow2 --release focal -c ubuntu
-    phenix image build ubuntu -o /phenix -c -x
-    ```
+- `ubuntu.qc2` - available from [sceptre-phenix-images](https://github.com/sandialabs/sceptre-phenix-images) 
 
-To run the model use the following commands
+To download `ubuntu.qc2`, install [ORAS](https://oras.land/docs/installation/), then run:
+
+```shell
+cd /phenix/images/
+oras pull ghcr.io/sandialabs/sceptre-phenix-images/ubuntu.qc2:latest
 ```
-phenix config create <path_to_topos>/helloworld.yaml
-phenix exp create helloworld -t helloworld
-phenix exp start helloworld
+
+To run the model, use the following commands:
+```shell
+phenix config create /phenix/topologies/helloworld.yaml
```

Comment:
`config create ...` not needed if you're using full paths in "experiment create ....".

Reply:
I guess based off the next note, maybe add a 3rd alternative using the `experiment create -t <full path> -s <full path>`?